stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -872,8 +872,8 @@ public class ChatClient internal constructor(
     }
 
```

Comment:
Did you verify that this param works as expected? From our docs I see:

```
// hard delete the message (works only server-side) 
await client.deleteMessage(messageID, true);
```

Strange, but several lines below I see that hard delete "Can be done client-side by users but be cautious this action is not recoverable"

Reply:
Docs are outdated

Reply:
Yep, that made me confused too.